sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/SumAggregation.java

```diff
@@ -27,14 +27,26 @@
  * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
  * at any time.
  */
-public final class SumAggregation implements Aggregation, AggregatorFactory {
+public final class SumAggregation implements AggregationExtension {
   private static final SumAggregation INSTANCE = new SumAggregation();
 
   public static Aggregation getInstance() {
     return INSTANCE;
   }
 
-  private SumAggregation() {}
+  private SumAggregation(ExemplarReservoirFactory reservoirFactory) {
+    this.reservoirFactory = reservoirFactory;
+  }
+
+  private SumAggregation() {
+    this(
+        ExemplarReservoirFactory.fixedSize(
+            Clock.getDefault(),
+            Runtime.getRuntime().availableProcessors(),
```

Comment:
Do we still think `Runtime.getRuntime().availableProcessors()` is best if the spec provides the new language:
>  the default size MAY be
the number of possible concurrent threads (e.g. numer of CPUs) to help reduce
contention. Otherwise, a default size of `1` SHOULD be used.

Reply:
I wrote the specification with this bit of code in mind....  What would you instead like to see?

The rationale here was a measurable difference in contention.